client/src/main/scala/com/aliyun/emr/rss/client/write/LifecycleManager.scala

```diff
@@ -931,76 +931,81 @@ class LifecycleManager(appId: String, val conf: RssConf) extends RpcEndpoint wit
   }
 
   /**
-   * reserve buffer with retry, retry on another node will cause slots to be inconsistent
+   * Reserve buffers with retry, retry on another node will cause slots to be inconsistent.
```

Comment:
`retry on another node will cause slots to be inconsistent` Want to know what it means.